docs/tutorials/essentials/part-4-using-data.md

```diff
@@ -498,13 +649,29 @@ Now, the root state looks like `{posts, users}`, matching the object we passed i
 
 ### Adding Authors for Posts
 
-Every post in our app was written by one of our users, and every time we add a new post, we should keep track of which user wrote that post.
+Every post in our app was written by one of our users, and every time we add a new post, we should keep track of which user wrote that post. This will need changes for both the Redux state and the `<AddPostForm>` component.
 
-To keep things simpler for this example, we'll update our `<AddPostForm>` component so that we can select a user from a dropdown list, and we'll include that user's ID as part of the post. Once our post objects have a user ID in them, we can use that to look up the user's name and show it in each individual post in the UI.
+First, we need to update the existing `Post` data type to include a `user: string` field that contains the user ID that created the post. We'll also update the existing post entries in `initialState` to have a `post.user` field with one of the example user IDs.
```

Comment:
I would prefer the key name to be `userId` instead of `user` since when I see `user` it sounds like it contains the full user object.

Reply:
Agreed, but it's not critical, and it would require some additional tweaks to the mock server setup as well.